environments/mainnet/permissions.yml

```diff
@@ -47,3 +47,19 @@
       - 'CREATE_VERSION_ROLE'
     - survey.aragonpm.eth:
       - 'CREATE_VERSION_ROLE'
+- address: '0xE04cAbcB24e11620Dd62bB99c396E76cEB578914'
```

Comment:
Did you separate it in on purpose from the previous item which is also for the cold wallet?

Reply:
Oh no, that was a mistake 😅. Fixed in ca78e09254c7286d9c0a2d37dfff3ff2eb8d15e1.